plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/AbstractTrinoCatalog.java

```diff
@@ -116,6 +116,15 @@ public abstract class AbstractTrinoCatalog
         implements TrinoCatalog
 {
     public static final String TRINO_CREATED_BY_VALUE = "Trino Iceberg connector";
+
+    public static class ViewProperties
```

Comment:
I like the idea of nested namespaces, but i am not convinced this is a pattern we used in this project.
Let's remove the class and let's rename the properties so it's clear they are for **iceberg views**.